--- /dev/null
+++ MODULE.bazel
@@ -0,0 +1,9 @@
+module(name = "sdflib")
+
+bazel_dep(name = "platforms", version = "1.0.0")
+bazel_dep(name = "rules_cc", version = "0.1.1")
+bazel_dep(name = "eigen", version = "3.4.0.bcr.3")
+bazel_dep(name = "zlib", version = "1.3.1.bcr.3")
+bazel_dep(name = "cereal", version = "1.3.2")
+bazel_dep(name = "glm", version = "1.0.1")
+bazel_dep(name = "spdlog", version = "1.15.2")
